api/URLSearchParams.json

```diff
@@ -89,16 +89,9 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": [
-              {
-                "version_added": "7.10.0"
-              },
-              {
-                "version_added": "7.5.0",
-                "partial_implementation": true,
-                "notes": "Only string as <code>init</code> object supported."
-              }
-            ],
```

Comment:
I removed the note because the information is covered by the later-added `USVString`, `record`, and `sequence` sub-properties.